### PR TITLE
Moon day

### PR DESCRIPTION
**Pull request by emilyPhee**
Add Moon Day Component
- installed Lune package 
- calculate Full Moon and New Moon phase
- display with grid

### CODE REVIEW

**Comment by emilyPhee** [comment]
Add Moon Day Component

 - installed Lune package
 - calculate Full Moon and New Moon phase
 - display with grid

**Comment by emilyPhee** [comment]
[Ashtanga: Moon day](https://trello.com/c/7OEpQayh/24-ashtanga-moon-day)

File: nextjs-client/src/components/YogaClassInfo/MoonDay/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    let date = new Date(year, specificMonth + 1, 0);
+    date.setHours(23);
+    date.setMinutes(59);
+    date.setSeconds(59);
+    return date;
+  };
+
+  const lastDayOfMonth = getLastDayOfMonth(currentYear, Number(month));
+  const firstDayOfMonth = new Date(`${currentYear}, ${Number(month) + 1}, 1`);
+  const lune = require('lune');
```
Comment (anthopark) [inline nextjs-client/src/components/YogaClassInfo/MoonDay/index.js]: change this to the import statement

**Review by anthopark — CHANGES_REQUESTED** [review]
🚀

**Review by anthopark — APPROVED** [review]

**Pull request merged**